vagrant-spk

```diff
@@ -105,6 +108,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provider :virtualbox do |vb, override|
     vb.cpus = cpus
     vb.memory = assign_ram_mb
+    vb.name = PROJECT_BASENAME + "_sandstorm"
```

Comment:
Darn.

I'm mildly concerned that setting vm.name to a non-random value will attempt to change the name every time you run a vagrant command, which would be annoying.  I guess it's not a particularly-bad behavior though, just slightly surprising.